sites/public/next.config.js

```diff
@@ -28,13 +28,6 @@ if (process.env.INCOMING_HOOK_BODY && process.env.INCOMING_HOOK_BODY.startsWith(
 const LISTINGS_QUERY = process.env.LISTINGS_QUERY || "/listings"
```

Comment:
update default here?

Reply:
I'm of two minds on that.  I recognize that `/listings/combined` should be the only one used for now, but given that this is just a temporary thing that will be reverted later I'm not sure that it makes sense to change the default behavior.  I'm generally more inclined to make those types of changes in configuration rather than code.

Reply:
Can we just remove the default in that case?

Reply:
Done